### PR TITLE
Add a guide list and a guide viewer to show guides

### PR DESCRIPTION
**Pull request by tinayuangao**
R: @jelbourn

### CODE REVIEW

File: src/app/pages/guide-list/guide-list.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<md-list class="docs-guide-list-item">
+  <md-list-item *ngFor="let guide of guideItems.getAllItems()">
+    <a [routerLink]="['/guide/', guide.id]">
+      {{guide.name}}
+    </a>
+  </md-list-item>
+</md-list>
```
Comment (jelbourn) [inline src/app/pages/guide-list/guide-list.html]: You should be able to do 
```html
<md-nav-list>
  <a md-list-item ...> ... </a>
</md-nav-list>
```

**Comment by tinayuangao** [comment]
Done. PTAL. Thanks for review!

**Comment by jelbourn** [comment]
@tinayuangao one lint error

**Comment by tinayuangao** [comment]
Fixed lint errors

**Pull request merged**